components/centraldashboard/frontend/components.html

```diff
@@ -1,4 +1,4 @@
-<base href="https://cdn.rawgit.com/download/polymer-cdn/2.6.0/lib/">
```

Comment:
I think I saw this in another place too, can you make sure rawgit isn't used anywhere?

Reply:
Done, there was one for webcomponents.js, removed and verified